### ISSUE TITLE
Official Gram Wallet application does not accept mnemonic.

### ISSUE THREAD

**Issue by ghost** [mercuryoio/tonlib-go#26]
Official Gram Wallet application does not accept `mnemonic` phrase generated by tonlib-go.
I user `CreatePrivateKey` method to generate keypair and `ExportPrivateKey` to export `mnemonic` phrase. (of course i use non-empty LocalPassword to generate and export keys)

For example:
```
Address: EQDe3p0RlLsrXMEzZUe55zO6S6rY3A82ZUFk74RwghZicM77
Public: PubIV5DzK0XwQjXyYp9DyFfN4TA57c5JHOzeXjY8ibIGx3an
Secret: YUR8dxOnbVAejYnQfhNdarSX4WdzxZLrfJ/HthdPN2Q=
Mnemonic: [cliff recipe stereo error churn ozone burger river arm crystal possible radar life focus margin stool taste village immune boost grape still moon winter]
```

I have no clue how to solve this issue. Perhaps this depends on my OS (Manjaro Linux x86_64) or `libtonjson` (I tried both: compiled `libtonjson` by myself and tried to use one from this repository. Anyway, I got the same result).
![1234](https://user-images.githubusercontent.com/17069684/68051172-76addd80-fcef-11e9-8d82-65bf28610700.png)

**Comment by akme** [comment]
Created an [Issue ](https://github.com/ton-blockchain/ton/issues/177) in ton-blockchain repo, possibly the problem takes its roots there.

**Comment by webresult** [comment]
It's fixed in v0.4.0 release. Unfortunately you can't use old keys. You should create new key without mnemonic password. It`ll work with desktop apps. 
Thank you!
